### PR TITLE
fix(cli): autogenerate examples for text and streaming

### PR DESCRIPTION
**Pull request by dsinghvi**
## Description
This PR updates the IR generation to generate examples for SSE, JSON streaming and Text. 

## Changes Made
- This is relevant because we have some logic that checks for autogenerated examples being present before sending code samples to FDR to render in Docs (when we don't have the examples present the code samples aren't sent). 

## Testing
- [x] Unit tests added/updated
- [x] Manual testing completed

### CODE REVIEW

File: packages/cli/api-importers/openapi/openapi-ir-parser/src/openapi/v3/extensions/getExamplesFromExtension.ts [hunk 1]
```diff
@@ -14,16 +14,15 @@ export function getExamplesFromExtension(
...
+        getExtension<RawSchemas.ExampleEndpointCallSchema[]>(
             operationObject,
             FernOpenAPIExtension.EXAMPLES
-            // RawSchemas.ExampleEndpointCallArraySchema,
-            // context.logger,
-            // [...operationContext.baseBreadcrumbs, `${operationContext.method} ${operationContext.path}`]
-        ) ?? [];
+        );
+    
+    context.logger.error(`exampleEndpointCalls ${JSON.stringify(exampleEndpointCalls)}`);
```
Comment (amckinney) [inline packages/cli/api-importers/openapi/openapi-ir-parser/src/openapi/v3/extensions/getExamplesFromExtension.ts:23]: We should get rid of this, right?

**Comment by dsinghvi** [comment]
https://github.com/fern-api/fern/blob/6d3eb48a4c090afe7a7627d960027a302c783ca2/packages/cli/register/src/ir-to-fdr-converter/convertPackage.ts#L73 is the docs logic

File: packages/cli/generation/ir-generator/src/examples/generator/generateSuccessEndpointExample.ts [hunk 2]
```diff
@@ -256,12 +268,71 @@ export function generateEndpointExample({
...
+                const { example, jsonExample } = generatedExample;
+                result.response = ExampleResponse.ok(ExampleEndpointSuccessResponse.body({ ...example, jsonExample }));
+                break;
+            }
+            case "streaming": {
+                const generatedExample = generateTypeReferenceExample({
+                    currentDepth: 0,
+                    maxDepth: 10,
+                    typeDeclarations,
+                    typeReference: TEXT_TYPE_REFERENCE,
```
Comment (amckinney) [inline packages/cli/generation/ir-generator/src/examples/generator/generateSuccessEndpointExample.ts:311]: Is this right?

Comment (dsinghvi) [reply packages/cli/generation/ir-generator/src/examples/generator/generateSuccessEndpointExample.ts:311]: oops good catch, will fix that

**Review by amckinney — DISMISSED** [review]

**Review by amckinney — APPROVED** [review]

**Review by fern-support — APPROVED** [review]

**Pull request merged**